2025-07-07 18:37:06 UTC
macOS-15.5-arm64-arm-64bit
Ren'Py 8.3.4.24120703

Early init took 0.03s
Loading error handling took 0.02s
Loading script took 0.15s
Loading save slot metadata took 0.03s
Loading persistent took 0.00s
Running init code took 0.03s
Loading analysis data took 0.01s
Analyze and compile ATL took 0.00s
Reloading save slot metadata took 0.01s
Index archives took 0.00s
Backing up script files to '/Users/toninodigiacomo/Library/RenPy/backups/Poker':
Dump and make backups took 0.00s
Cleaning cache took 0.00s
Making clean stores took 0.00s
Initial gc took 0.02s
DPI scale factor: 1.000000
Creating interface object took 0.00s
Cleaning stores took 0.00s
Init translation took 0.02s
Build styles took 0.00s
Load screen analysis took 0.00s
Analyze screens took 0.01s
Save screen analysis took 0.01s
Prepare screens took 0.04s
Save pyanalysis. took 0.01s
Save bytecode. took 0.02s
Running _start took 0.00s
Interface start took 0.35s

Initializing gl2 renderer:
primary display bounds: (0, 0, 3840, 2160)
swap interval: 1 frames
Windowed mode.
Vendor: "b'Apple'"
Renderer: b'Apple M1'
Version: b'2.1 Metal - 89.4'
Display Info: None
Screen sizes: virtual=(2560, 1440) physical=(2385, 1342) drawable=(2385, 1342)
Maximum texture size: 4096x4096
INFO:renpy_warp_service:service thread started. periodically scanning ports for warp server

Full traceback:
  File "renpy/common/_layout/screen_load_save.rpym", line 35, in script
    $ ui.interact()
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/ast.py", line 827, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/_layout/screen_load_save.rpym", line 35, in <module>
    $ ui.interact()
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 2215, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 2745, in interact_core
    root_widget.visit_all(lambda d : d.per_interact())
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  [Previous line repeated 1 more time]
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/screen.py", line 480, in visit_all
    callback(self)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 2745, in <lambda>
    root_widget.visit_all(lambda d : d.per_interact())
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/screen.py", line 491, in per_interact
    self.update()
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/screen.py", line 697, in update
    self.screen.function(**self.scope)
  File "game/screens.rpy", line 585, in execute
    screen save():
  File "game/screens.rpy", line 585, in execute
    screen save():
  File "game/screens.rpy", line 589, in execute
    use file_slots(_("Save"))
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/parameter.py", line 503, in evaluate
    value = renpy.python.py_eval(v, locals=scope)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/python.py", line 1209, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/python.py", line 1202, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/screens.rpy", line 589, in <module>
    use file_slots(_("Save"))
TypeError: 'RevertableDict' object is not callable

While running game code:
  File "game/screens.rpy", line 585, in execute
    screen save():
  File "game/screens.rpy", line 585, in execute
    screen save():
  File "game/screens.rpy", line 589, in execute
    use file_slots(_("Save"))
  File "game/screens.rpy", line 589, in <module>
    use file_slots(_("Save"))
TypeError: 'RevertableDict' object is not callable
While handling exception:
Traceback (most recent call last):
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/execution.py", line 599, in run
    node.execute()
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/ast.py", line 827, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/_layout/screen_load_save.rpym", line 35, in <module>
    $ ui.interact()
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 2215, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 2745, in interact_core
    root_widget.visit_all(lambda d : d.per_interact())
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  [Previous line repeated 1 more time]
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/screen.py", line 480, in visit_all
    callback(self)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 2745, in <lambda>
    root_widget.visit_all(lambda d : d.per_interact())
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/screen.py", line 491, in per_interact
    self.update()
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/screen.py", line 697, in update
    self.screen.function(**self.scope)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 2716, in __call__
    self.execute(context)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 2654, in execute
    SLBlock.execute(self.const_ast, context) # type: ignore
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 500, in execute
    i.execute(context)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 2117, in execute
    args, kwargs = self.args.evaluate(context.scope)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/parameter.py", line 503, in evaluate
    value = renpy.python.py_eval(v, locals=scope)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/python.py", line 1209, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/python.py", line 1202, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/screens.rpy", line 589, in <module>
    use file_slots(_("Save"))
TypeError: 'RevertableDict' object is not callable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/error.py", line 140, in report_exception
    renpy.game.invoke_in_new_context(
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/game.py", line 303, in invoke_in_new_context
    return callable(*args, **kwargs)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/error.py", line 48, in call_exception_screen
    return renpy.ui.interact(mouse="screen", type="screen", suppress_overlay=True, suppress_underlay=True)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 2215, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 2745, in interact_core
    root_widget.visit_all(lambda d : d.per_interact())
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/screen.py", line 480, in visit_all
    callback(self)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 2745, in <lambda>
    root_widget.visit_all(lambda d : d.per_interact())
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/screen.py", line 491, in per_interact
    self.update()
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/screen.py", line 697, in update
    self.screen.function(**self.scope)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 2716, in __call__
    self.execute(context)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 2654, in execute
    SLBlock.execute(self.const_ast, context) # type: ignore
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 500, in execute
    i.execute(context)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 1098, in execute
    i.execute(ctx)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 1098, in execute
    i.execute(ctx)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 1098, in execute
    i.execute(ctx)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 944, in execute
    positional = eval(positional_exprs, context.globals, context.scope)
  File "renpy/common/_errorhandling.rpym", line 716, in <module>
    label _("An exception has occurred.") text_size gui._scale(40)
TypeError: 'RevertableDict' object is not callable

Full traceback:
  File "game/script.rpy", line 740, in script
    if result == 'allin':
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/ast.py", line 827, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 777, in <module>
    # Affiche le screen d’action et attend le choix du joueur
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/exports/statementexports.py", line 194, in pause
    rv = renpy.ui.interact(mouse='pause', type='pause', roll_forward=roll_forward, pause=delay, pause_modal=modal)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 2215, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 3286, in interact_core
    rv = root_widget.event(ev, x, y, 0)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/layout.py", line 1297, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/behavior.py", line 554, in event
    rv = run(action)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/behavior.py", line 401, in run
    return action(*args, **kwargs)
  File "renpy/common/00gamemenu.rpy", line 130, in _invoke_game_menu
    renpy.call_in_new_context('_game_menu')
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/game.py", line 377, in call_in_new_context
    return renpy.execution.run_context(False)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/execution.py", line 958, in run_context
    context.run()
  File "renpy/common/_layout/screen_load_save.rpym", line 35, in script
    $ ui.interact()
  File "lib/python3.9/future/utils/__init__.py", line 444, in raise_
  File "renpy/common/_layout/screen_load_save.rpym", line 35, in script
    $ ui.interact()
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/ast.py", line 827, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/_layout/screen_load_save.rpym", line 35, in <module>
    $ ui.interact()
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 2215, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 2745, in interact_core
    root_widget.visit_all(lambda d : d.per_interact())
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  [Previous line repeated 1 more time]
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/screen.py", line 480, in visit_all
    callback(self)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 2745, in <lambda>
    root_widget.visit_all(lambda d : d.per_interact())
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/screen.py", line 491, in per_interact
    self.update()
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/screen.py", line 697, in update
    self.screen.function(**self.scope)
  File "game/screens.rpy", line 585, in execute
    screen save():
  File "game/screens.rpy", line 585, in execute
    screen save():
  File "game/screens.rpy", line 589, in execute
    use file_slots(_("Save"))
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/parameter.py", line 503, in evaluate
    value = renpy.python.py_eval(v, locals=scope)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/python.py", line 1209, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/python.py", line 1202, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/screens.rpy", line 589, in <module>
    use file_slots(_("Save"))
TypeError: 'RevertableDict' object is not callable

While running game code:
  File "game/script.rpy", line 740, in script
    if result == 'allin':
  File "game/script.rpy", line 777, in <module>
    # Affiche le screen d’action et attend le choix du joueur
  File "renpy/common/00gamemenu.rpy", line 130, in _invoke_game_menu
    renpy.call_in_new_context('_game_menu')
  File "game/screens.rpy", line 585, in execute
    screen save():
  File "game/screens.rpy", line 585, in execute
    screen save():
  File "game/screens.rpy", line 589, in execute
    use file_slots(_("Save"))
  File "game/screens.rpy", line 589, in <module>
    use file_slots(_("Save"))
TypeError: 'RevertableDict' object is not callable
While handling exception:
Traceback (most recent call last):
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/execution.py", line 629, in run
    if renpy.display.error.report_exception(short, full, traceback_fn):
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/error.py", line 140, in report_exception
    renpy.game.invoke_in_new_context(
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/game.py", line 303, in invoke_in_new_context
    return callable(*args, **kwargs)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/error.py", line 48, in call_exception_screen
    return renpy.ui.interact(mouse="screen", type="screen", suppress_overlay=True, suppress_underlay=True)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 2215, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 2745, in interact_core
    root_widget.visit_all(lambda d : d.per_interact())
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/screen.py", line 480, in visit_all
    callback(self)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 2745, in <lambda>
    root_widget.visit_all(lambda d : d.per_interact())
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/screen.py", line 491, in per_interact
    self.update()
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/screen.py", line 697, in update
    self.screen.function(**self.scope)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 2716, in __call__
    self.execute(context)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 2654, in execute
    SLBlock.execute(self.const_ast, context) # type: ignore
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 500, in execute
    i.execute(context)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 1098, in execute
    i.execute(ctx)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 1098, in execute
    i.execute(ctx)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 1098, in execute
    i.execute(ctx)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 944, in execute
    positional = eval(positional_exprs, context.globals, context.scope)
  File "renpy/common/_errorhandling.rpym", line 716, in <module>
    label _("An exception has occurred.") text_size gui._scale(40)
TypeError: 'RevertableDict' object is not callable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/execution.py", line 599, in run
    node.execute()
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/ast.py", line 827, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 777, in <module>
    # Affiche le screen d’action et attend le choix du joueur
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/exports/statementexports.py", line 194, in pause
    rv = renpy.ui.interact(mouse='pause', type='pause', roll_forward=roll_forward, pause=delay, pause_modal=modal)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 2215, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 3286, in interact_core
    rv = root_widget.event(ev, x, y, 0)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/layout.py", line 1297, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/behavior.py", line 554, in event
    rv = run(action)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/behavior.py", line 401, in run
    return action(*args, **kwargs)
  File "renpy/common/00gamemenu.rpy", line 130, in _invoke_game_menu
    renpy.call_in_new_context('_game_menu')
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/game.py", line 377, in call_in_new_context
    return renpy.execution.run_context(False)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/execution.py", line 958, in run_context
    context.run()
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/execution.py", line 634, in run
    reraise(exc_info[0], exc_info[1], exc_info[2])
  File "lib/python3.9/future/utils/__init__.py", line 444, in raise_
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/execution.py", line 599, in run
    node.execute()
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/ast.py", line 827, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/_layout/screen_load_save.rpym", line 35, in <module>
    $ ui.interact()
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 2215, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 2745, in interact_core
    root_widget.visit_all(lambda d : d.per_interact())
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  [Previous line repeated 1 more time]
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/screen.py", line 480, in visit_all
    callback(self)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 2745, in <lambda>
    root_widget.visit_all(lambda d : d.per_interact())
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/screen.py", line 491, in per_interact
    self.update()
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/screen.py", line 697, in update
    self.screen.function(**self.scope)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 2716, in __call__
    self.execute(context)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 2654, in execute
    SLBlock.execute(self.const_ast, context) # type: ignore
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 500, in execute
    i.execute(context)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 2117, in execute
    args, kwargs = self.args.evaluate(context.scope)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/parameter.py", line 503, in evaluate
    value = renpy.python.py_eval(v, locals=scope)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/python.py", line 1209, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/python.py", line 1202, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/screens.rpy", line 589, in <module>
    use file_slots(_("Save"))
TypeError: 'RevertableDict' object is not callable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/error.py", line 140, in report_exception
    renpy.game.invoke_in_new_context(
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/game.py", line 303, in invoke_in_new_context
    return callable(*args, **kwargs)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/error.py", line 48, in call_exception_screen
    return renpy.ui.interact(mouse="screen", type="screen", suppress_overlay=True, suppress_underlay=True)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 2215, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 2745, in interact_core
    root_widget.visit_all(lambda d : d.per_interact())
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/screen.py", line 480, in visit_all
    callback(self)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 2745, in <lambda>
    root_widget.visit_all(lambda d : d.per_interact())
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/screen.py", line 491, in per_interact
    self.update()
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/screen.py", line 697, in update
    self.screen.function(**self.scope)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 2716, in __call__
    self.execute(context)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 2654, in execute
    SLBlock.execute(self.const_ast, context) # type: ignore
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 500, in execute
    i.execute(context)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 1098, in execute
    i.execute(ctx)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 1098, in execute
    i.execute(ctx)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 1098, in execute
    i.execute(ctx)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/sl2/slast.py", line 944, in execute
    positional = eval(positional_exprs, context.globals, context.scope)
  File "renpy/common/_errorhandling.rpym", line 716, in <module>
    label _("An exception has occurred.") text_size gui._scale(40)
TypeError: 'RevertableDict' object is not callable

Full traceback:
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/bootstrap.py", line 359, in bootstrap
    renpy.main.main()
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/main.py", line 652, in main
    run(restart)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/main.py", line 148, in run
    renpy.execution.run_context(True)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/execution.py", line 958, in run_context
    context.run()
  File "game/script.rpy", line 740, in script
    if result == 'allin':
  File "lib/python3.9/future/utils/__init__.py", line 444, in raise_
  File "game/script.rpy", line 740, in script
    if result == 'allin':
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/ast.py", line 827, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 777, in <module>
    # Affiche le screen d’action et attend le choix du joueur
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/exports/statementexports.py", line 194, in pause
    rv = renpy.ui.interact(mouse='pause', type='pause', roll_forward=roll_forward, pause=delay, pause_modal=modal)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 2215, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 3286, in interact_core
    rv = root_widget.event(ev, x, y, 0)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/layout.py", line 1297, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/behavior.py", line 554, in event
    rv = run(action)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/behavior.py", line 401, in run
    return action(*args, **kwargs)
  File "renpy/common/00gamemenu.rpy", line 130, in _invoke_game_menu
    renpy.call_in_new_context('_game_menu')
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/game.py", line 377, in call_in_new_context
    return renpy.execution.run_context(False)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/execution.py", line 958, in run_context
    context.run()
  File "renpy/common/_layout/screen_load_save.rpym", line 35, in script
    $ ui.interact()
  File "lib/python3.9/future/utils/__init__.py", line 444, in raise_
  File "renpy/common/_layout/screen_load_save.rpym", line 35, in script
    $ ui.interact()
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/ast.py", line 827, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/_layout/screen_load_save.rpym", line 35, in <module>
    $ ui.interact()
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 2215, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 2745, in interact_core
    root_widget.visit_all(lambda d : d.per_interact())
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  [Previous line repeated 1 more time]
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/screen.py", line 480, in visit_all
    callback(self)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/core.py", line 2745, in <lambda>
    root_widget.visit_all(lambda d : d.per_interact())
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/screen.py", line 491, in per_interact
    self.update()
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/display/screen.py", line 697, in update
    self.screen.function(**self.scope)
  File "game/screens.rpy", line 585, in execute
    screen save():
  File "game/screens.rpy", line 585, in execute
    screen save():
  File "game/screens.rpy", line 589, in execute
    use file_slots(_("Save"))
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/parameter.py", line 503, in evaluate
    value = renpy.python.py_eval(v, locals=scope)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/python.py", line 1209, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "/Volumes/INTERNAL_NVM/Games/RenPy SDK/renpy/python.py", line 1202, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/screens.rpy", line 589, in <module>
    use file_slots(_("Save"))
TypeError: 'RevertableDict' object is not callable

While running game code:
  File "game/script.rpy", line 740, in script
    if result == 'allin':
  File "game/script.rpy", line 740, in script
    if result == 'allin':
  File "game/script.rpy", line 777, in <module>
    # Affiche le screen d’action et attend le choix du joueur
  File "renpy/common/00gamemenu.rpy", line 130, in _invoke_game_menu
    renpy.call_in_new_context('_game_menu')
  File "game/screens.rpy", line 585, in execute
    screen save():
  File "game/screens.rpy", line 585, in execute
    screen save():
  File "game/screens.rpy", line 589, in execute
    use file_slots(_("Save"))
  File "game/screens.rpy", line 589, in <module>
    use file_slots(_("Save"))
TypeError: 'RevertableDict' object is not callable
